composer.json

```diff
@@ -31,11 +31,11 @@
   "require-dev": {
     "phpunit/phpunit": "4.8.*",
     "squizlabs/php_codesniffer": "2.*",
-    "phpdocumentor/phpdocumentor": "2.*"
+    "phpdocumentor/phpdocumentor": "2.8.*"
   },
   "autoload": {
     "psr-4": {
-      "Google\\Cloud\\": "src"
+      "Gcloud\\": "src"
```

Comment:
After spending a little more time looking at what some of the other gcloud-* projects have done I thought it would make sense to have the namespace match the branding of 'gcloud' found on our doc sites and elsewhere.

I would be open to reverting back to the `Google\Cloud` namespace if that seems more fitting, especially since there is already traction in that direction.